### ISSUE TITLE
Performance: Widgets

### ISSUE THREAD

**Issue by haroldangenent** [trendwerk/trendpress#742]
While TrendPress has no sidebars, WordPress [still initializes it's widgets](https://github.com/WordPress/WordPress/blob/master/wp-includes/default-filters.php#L443).

We could [remove them](http://stackoverflow.com/questions/9219373/removing-default-widgets-in-wordpress) and gain some performance. Oversimplified:

```
remove_action('init', 'wp_widgets_init', 1);
```

_Probably better as a separate library, but just gathering opinions here._

**Comment by luukdv** [comment]
Sounds good! I agree about implementing it as a library.

**Comment by sboerrigter** [comment]
I agree. I think it makes sense to add this in the theme since the theme does not support widgets. But as a seperate library is fine as well.
